platform/docs/docs/configuration/theme-configuration.md

```diff
@@ -0,0 +1,287 @@
+# Theme Configuration
+When adding new theme extendible configuration items, please document
+them here.  See [Theme Configuration with Config Point](#configPoint) on how to modify certain types of
+configuration values using the config-point defintions.
+
+## Hanging Protocols
+It is possible to customize the available hanging protocols by defining them
+in a new theme file OR by defining the hanging protocols in a custom mode.
+If done correctly, the mode defined hanging protocols are automatically
+applied when using a particular view mode, allowing for further customization
+by theme files.
+
+The default hanging protocols for the cornerstone mode is defined in
+themeProtocolProvider.js,  in a configuration point named
+`ThemeProtocols.protocols`  To define a new
+hanging protocol, it is possible to simply extend the protocols list with
+a new definition, for example, in the file mgHP.json5, the following definition
+would add a new hanging protocol:
+```js
+{
+  ThemeProtocols: {
+    protocols: {
```

Comment:
Why this is an object instead of an array? I believe protocols should be an array of prtocols. Is this how config-points work internaly?

```
{
 ThemeProtocols: {
   protocols:[
     {
       id: 'MG'
       ....
     },
     {
       id:'prostate',
       ...
     }
   ]
 }
}
```

Reply:
protocols is an array as originally declared, but there is an object based syntax that can be used to extend the array.  If you could do something like:
[ insertAtEnd: newValue ]
then it would work, but because that syntax is kind of ugly, the alternate syntax here works.  In fact, I have a syntax
defined that allows adding them as array elements, but it is kind of long/ugly:
[ {configOperation: 'insert', value: { ... MG protocol definition } }, ... ]